tests/e2e/example.test.e2e.js

```diff
@@ -44,6 +44,18 @@ test('Test Login Form', async ({page}, workerInfo) => {
   save_visual(page);
 });
 
+test('Test Login Form OpenID navigation', async ({page}, workerInfo) => {
+  const response = await page.goto('/user/login');
+  expect(response?.status()).toBe(200); // Status OK
+
+  await page.click('form .button.ui.openid');
```

Comment:
```suggestion
  await page.getByRole('link', {name: 'Sign in with OpenID'}).click();
```

CSS selectors are strongly discouraged in playwright. Select by button text instead.